CHANGELOG.md

```diff
@@ -1,17 +1,25 @@
-## [1.0.11](https://github.com/Path-Check/covid-safe-paths/compare/v0.5.4...1.0.11) (2020-05-13)
+## [1.0.13](https://github.com/sparks2209/covid-safe-paths/compare/1.0.12...1.0.13) (2020-05-21)
```

Comment:
Undo these changes to this file

Reply:
Do mean to keep CHANGELOG.md file empty? or just to deleted above added line.